src/app/login/login.component.spec.ts

```diff
@@ -0,0 +1,30 @@
+import {ComponentFixture, TestBed} from '@angular/core/testing';
+
+import {LoginComponent} from './login.component';
+import {Spectator} from "@ngneat/spectator/jest";
+
+describe('LoginComponent', () => {
+  let component: LoginComponent;
+  let fixture: ComponentFixture<LoginComponent>;
+  let spectator: Spectator<LoginComponent>;
```

Comment:
If you want to use the spectator create a component with the componentFactory and in the beforeEach initialize the spectator with it. The spectator makes it easier to query elements and so on.
```
const createComponent = createComponentFactory({
    detectChanges: false,
    component: LoginComponent,
  });

  beforeEach(() => {
    spectator = createComponent();
    fixture = spectator.fixture;
    component = spectator.component;
  })
```